benchmark/lib/Streamly/Benchmark/Prelude.hs

```diff
@@ -68,6 +68,9 @@ import Control.Applicative (liftA2)
 import Control.DeepSeq (NFData(..))
 import Control.Exception (try)
 import Data.Functor.Identity (Identity)
+#if !(MIN_VERSION_base(4,11,0))
+import Data.Semigroup (Semigroup((<>)))
+#endif
```

Comment:
This has to be removed. This means GHC versions <= 8.2.x which we are not supporting any more.